src/main/java/org/opentripplanner/transit/raptor/rangeraptor/path/ReversePathMapper.java

```diff
@@ -142,16 +139,15 @@ else if(fromStopArrival.arrivedByAccess()) {
 
         // No need to time-shift the egress leg, this is done when stopArrival is created
         return new EgressPathLeg<>(
-            egress,
-            accessArrival.stop(),
-            targetFromTime,
-            targetToTime,
-            RaptorCostConverter.toOtpDomainCost(accessArrival.cost())
+                egress,
+                accessArrival.stop(),
+                targetFromTime,
+                targetToTime
         );
     }
 
-    private int domainCost(ArrivalView<T> from) {
+    private static int legCost(ArrivalView<?> from) {
         if(from.cost() == -1) { return -1; }
-        return RaptorCostConverter.toOtpDomainCost(from.cost() - from.previous().cost());
+        return from.cost() - from.previous().cost();
     }
```

Comment:
As far as I can tell this is duplicated in `ForwardPathMapper` (`transitCost`).

Reply:
Yes, you are correct this in not completely DRY, but creating some sort of common utility for the two mappers  seamed to a bit overkill for just this function - so I went for the duplication. I do not not want calls between the two mappers since that increases complexity. I will leave this as is.